apptools/persistence/spickle.py

```diff
@@ -19,10 +19,10 @@
 import pickle
 import struct
 from pickle import Pickler, Unpickler, dumps, BUILD, NEWOBJ, REDUCE, \
-     MARK, OBJ, INST, BUILD, TupleType, PicklingError, GLOBAL, \
+     MARK, OBJ, INST, BUILD, PicklingError, GLOBAL, \
      EXT1, EXT2, EXT4, _extension_registry, _keep_alive
```

Comment:
Since this module doesn't work on Python 3, I would recommend checking for Python 3 explicitly and raising an `ImportError` with a message that explains this.